onnx/backend/test/case/model/expand.py

```diff
@@ -34,32 +34,32 @@ def make_graph(node, input_shape, shape_shape, output_shape):  # type: (onnx.hel
         node = onnx.helper.make_node(
             'Expand', ['X', 'shape'], ['Y'], name='test')
         input_shape = [1, 3, 1]
-        x = np.ones(input_shape)
+        x = np.ones(input_shape, dtype=np.float32)
 
         #1st testcase
         shape = np.array([3, 1])
```

Comment:
Here, you also need to turn them into np.int64. Otherwise, on 32-bit machine, we will generate np.int32 instead.

Reply:
yeah you are right

Reply:
Also do you need to update the generated data files too?